src/main/java/no/nav/pto/veilarbportefolje/client/ClientConfig.java

```diff
@@ -32,6 +30,15 @@ public AktorOppslagClient aktorOppslagClient(SystemUserTokenProvider systemUserT
         return new CachedAktorOppslagClient(aktorOppslagClient);
     }
 
+    @Bean
+    public PdlClient pdlClient(SystemUserTokenProvider systemUserTokenProvider){
+        return new PdlClientImpl(
+                createServiceUrl("pdl-api", "default", false),
```

Comment:
maybe appName and namespace can be constants, so its easier later to change them